wezterm-gui/src/main.rs

```diff
@@ -43,6 +43,7 @@ mod markdown;
 mod overlay;
 mod quad;
 mod renderstate;
+mod ricalc;
```

Comment:
Let's use a more verbose name for this, as otherwise I'm sure that I won't remember what this is in a month or two :)

```suggestion
mod resize_increment_calculator;
```